dotcom-rendering/src/layouts/PictureLayout.tsx

```diff
@@ -573,22 +572,6 @@ export const PictureLayout = (props: WebProps | AppsProps) => {
 								/>
 							</ArticleContainer>
 						</GridItem>
-						{isWeb && (
```

Comment:
Picture Layouts aren't best suited to showing this component. We can leave out pages with the Picture Layout from the AB test and still hit sample sizes